src/hooks/useDisableScroll/__tests__/useDisableScroll.jest.ts

```diff
@@ -0,0 +1,45 @@
+import { renderHook } from "@testing-library/react-hooks";
+import useDisableScroll from "..";
+
+describe("useDisableScroll", () => {
+  let addEventListenerSpy: jest.SpyInstance;
```

Comment:
These tests are bit problematic because they are depend on the hook implementation. Ideally the tests should check if we succeed to call enableScroll and disableScroll and if it is affect the user ability to scroll.

Reply:
Will fix in other pr 🙏